test/videojs-contrib-hls.test.js

```diff
@@ -1133,10 +1133,10 @@ QUnit.test('reloads out-of-date live playlists when switching variants', functio
 });
 
 QUnit.test('if withCredentials global option is used, withCredentials is set on the XHR object', function() {
-  let hlsOptions = videojs.options.hls;
+  let hlsOptions = videojs.Hls.options;
 
   this.player.dispose();
-  videojs.options.hls = {
```

Comment:
It would I think I have a more backwards compatible solution, I will submit it shortly